### PR TITLE
Add Arduino IDE library badge to README

### PR DESCRIPTION
**Pull request by mfalkvidd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,4 +16,4 @@ Current build status of development branch: [![Build Status](https://ci.mysensor
 
 Current build status of master branch (nightly build of Arduino IDE): [![Build Status](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/master/badge/icon)](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/master/)
 
-Current build status of development branch (nightly build of Arduino IDE): [![Build Status](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/development/badge/icon)](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/development/)
\ No newline at end of file
+Current build status of development branch (nightly build of Arduino IDE): [![Build Status](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/development/badge/icon)](https://ci.mysensors.org/job/MySensors-nightly-IDE/job/MySensors/job/development/)
```
Comment (fallberg) [inline README.md]: What is the actual change here?

Comment (mfalkvidd) [reply README.md]: Good question. I tried to find it when I made the commit, but was unable to find any difference. My best guess is that vi made some automatic white space change. https://github.com/mysensors/MySensors/pull/1254/files/b31df35392a8f2625c1820050e749eed11a1f27c?w=1 (option=ignore whitespace changes) says that the line in question was not changed.

**Comment by mfalkvidd** [comment]
hexdump.exe -C README.md, old version:
```
00000490  2f 4d 79 53 65 6e 73 6f  72 73 2f 6a 6f 62 2f 64  |/MySensors/job/d|
000004a0  65 76 65 6c 6f 70 6d 65  6e 74 2f 29              |evelopment/)|
```
new version:
```
00000540  2f 6a 6f 62 2f 64 65 76  65 6c 6f 70 6d 65 6e 74  |/job/development|
00000550  2f 29 0a                                          |/).|
```
So my editor seems to have added 0x0a. I was able to remove it by entering binary and hex mode in vi.

**Comment by fallberg** [comment]
Classic CR vs CR+LF

**Pull request merged**